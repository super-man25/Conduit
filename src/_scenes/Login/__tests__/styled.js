import React from 'react';
import {
  HideMe,
  ForgotLink,
  Content,
  LoginFooter,
  LogoImg
} from '../components/styled';
import edLogoImage from '_images/eventdynamiclogo.svg';
import renderer from 'react-test-renderer';

describe('Login styled components', () => {
  it('<HideMe /> should render correctly', () => {
    const tree = renderer.create(<HideMe />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('<ForgotLink /> should render correctly', () => {
    const tree = renderer.create(<ForgotLink />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('<Content /> should render correctly', () => {
    const tree = renderer.create(<Content />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('<LoginFooter /> should render correctly', () => {
    const tree = renderer.create(<LoginFooter />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('<LogoImg /> should render correctly', () => {
    const tree = renderer
      .create(<LogoImg src={edLogoImage} alt="Some alt text" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
