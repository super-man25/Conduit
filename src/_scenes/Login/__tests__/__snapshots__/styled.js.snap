// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login styled components <Content /> should render correctly 1`] = `
.c0 {
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;

exports[`Login styled components <ForgotLink /> should render correctly 1`] = `
.c0 {
  cursor: pointer;
  color: #D1D1D1;
  font-size: 12px;
  font-weight: normal;
}

.c0:hover {
  color: #070707;
}

<span
  className="c0"
/>
`;

exports[`Login styled components <HideMe /> should render correctly 1`] = `
.c0 {
  display: none;
  margin: 0;
  padding: 0;
}

<div
  className="c0"
/>
`;

exports[`Login styled components <LoginFooter /> should render correctly 1`] = `
.c0 {
  padding: 40px;
  background-color: #F2F2F2;
  font-size: 10px;
  line-height: 130%;
}

<div
  className="c0"
/>
`;

exports[`Login styled components <LogoImg /> should render correctly 1`] = `
.c0 {
  display: block;
  margin: auto;
  padding: 40px;
  padding-top: 10px;
  width: 40%;
}

<img
  alt="Some alt text"
  className="c0"
  src="eventdynamiclogo.svg"
/>
`;
